WHCFC_Frontend/src/index.html

```diff
@@ -6,8 +6,15 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="assets/team_logo_no_bg.png">
+  <script async src="https://www.googletagmanager.com/gtag/js?id={{ G-1FGQ64CEM1 }}"></script>
```

Comment:
It's usually not good practice to add this code in the index.html file. In Angular, you could use the app.component html file